python/lsst/daf/butler/cli/opt/options.py

```diff
@@ -122,6 +122,13 @@ def makeCollectionTypes(context, param, value):
                                     " JSON log records will be written, else formatted text log records"
                                     " will be written. This file can exist and records will be appended.")
 
+log_label_option = MWOptionDecorator("--log-label",
+                                     default=None,
+                                     multiple=True,
+                                     callback=split_kv,
+                                     type=str,
+                                     help="Keyword=value pairs to add to JSON log records.")
```

Comment:
I think it's not only about JSON, should we say "add to MDC of log records"?

Reply:
Okay. We don't yet allow people to specify the template for long-log but in the future we could do so and that would make the content of these labels available to the formatted logs.